File: Java/program69.java
import java.util.*;

class Digits
{
    public int CountFrequency(int iNo)
    {
        int iCnt = 0, iDigit = 0;
        if(iNo < 0)
        {
            iNo = -iNo;
        }
       
        while(iNo != 0) 
        {
            iDigit = iDigit % 10;
            if(iDigit == 5)
            {
                iCnt++;
            }
            iNo = iNo /10;
        }

        return iCnt;
    }

}
class program69
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iRet = 0;

        System.out.println("Enter number : ");
        int iValue = sobj.nextInt();

        Digits dobj =new Digits();

        iRet = dobj.CountFrequency(iValue);

        System.out.println("THe frequency of 5 is :"+iRet);

    }
}

File: Python/program69.py
class Digit:
    _No = 0
    
    def __init__(self, iNo):
        self._No = iNo
           
    def Count_digit_Frequency(self):
        
        if(self._No < 0):
            self._No = -self._No
        iCount = 0
        while(self._No > 0):
            iDigit = self._No % 10
            if(iDigit == 5):
                iCount += 1
            self._No = self._No // 10
            
        return iCount
    
def main(): 
    
    iValue = int (input("Enter number : "))
        
    sobj = Digit(iValue)
    iRet = sobj.Count_digit_Frequency()
    
    print("Frequency of 5 is  ",iRet)

if __name__ == "__main__":
    main()